cpp/src/Ice/Properties.cpp

```diff
@@ -319,11 +321,18 @@ Ice::Properties::setProperty(string_view key, string_view value)
     // Check if the property is in an Ice property prefix. If so, check that it's a valid property.
     if (auto propertyArray = findIcePropertyArray(key))
     {
+        if (propertyArray->isServicePrefix &&
+            (_servicePrefix != propertyArray->name && propertyArray->name != string{"IceStorm"}))
+        {
```

Comment:
Can't we have multiple servicePrefixes, instead of hardcoding IceStorm here?

Reply:
IceGrid doesn't set any IceStorm properties.

Reply:
I'm going to update to a list.

Reply:
I've updated to take list and removed the hard-coded "IceStorm".